src/base_authentication_app_skeleton/src/emails/templates/password_reset_request_email/html.ecr.ecr

```diff
@@ -1,3 +1,8 @@
 <h1>Please reset your password</h1>
 
+<%- if api_only? -%>
+<a href="<%="<"%>%= Api::PasswordResets::New.url(@user.id, @token) %>">Reset password</a>
```

Comment:
I think the conditional should be removed and this and the other password reset stuff should be moved to the browser specific auth folder.

We can tackle password reset later when we have really nailed down how it should work. Does that sound good to you?

Reply:
Bump

Reply:
Ah yes, I had totally forgotten about this. Thanks for bringing it to my attention.